static/app/utils/theme.tsx

```diff
@@ -95,6 +95,18 @@ const darkColors = {
   pink100: 'rgba(250, 51, 150, 0.13)',
 };
 
+const lightShadows = {
+  dropShadowLightest: '0 0 2px rgba(43, 34, 51, 0.04)',
+  dropShadowLight: '0 1px 4px rgba(43, 34, 51, 0.04)',
+  dropShadowHeavy: '0 4px 24px rgba(43, 34, 51, 0.12)',
+};
+
+const darkShadows = {
+  dropShadowLightest: '0 0 2px rgba(10, 8, 12, 0.2)',
+  dropShadowLight: '0 1px 4px rgba(10, 8, 12, 0.2)',
+  dropShadowHeavy: '0 4px 24px rgba(10, 8, 12, 0.36)',
+};
+
```

Comment:
Adding a new set of box shadow colors for dark mode. The current colors are almost invisible on dark backgrounds.